web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.html

```diff
@@ -14,9 +14,9 @@
         </div>
         <ul role="tablist" class="nav nav-tabs justify-content-start" *ngIf="!searchBox.value" id="explorer">
             <li class="nav-item">
-                <button [ngClass]="{'active' : !showLibrary}" role="tab" (click)="switchToProject()" class="nav-link"
-                    id="ngb-tab-0" aria-disabled="false" [attr.aria-expanded]="!showLibrary ? 'true' : 'false'"
-                    [attr.aria-controls]="!showLibrary ? 'ngb-tab-0-panel' : null">{{'project'
+                <button [ngClass]="{'active' : showProject}" role="tab" (click)="switchToProject()" class="nav-link"

```

Comment:
Instead of 

-  `showProject` i would then go with `currentActiveTab===ActiveTab.project`
-  `showLibrary` i would then go with `currentActiveTab===ActiveTab.library`
-  `showRecycleBin` i would then go with `currentActiveTab===ActiveTab.recylceBin`